src/Views/Search/Search.js

```diff
@@ -21,6 +21,7 @@ const Search = ({ loggedUser }) => {
   const handleSubmit = async (e) => {
     e.preventDefault();
     setStatus("Searching...");
+    setBooks([]);
     setPage(1);
```

Comment:
```suggestion
    const handleSubmit = async (e) => {
    e.preventDefault();
    setStatus("Searching...");
    setBooks([]);
    setPage(1);
    const res = await searchAllBooks({
      name: searchTitle,
      location: searchLocation,
    });
    if (res.error) setStatus(res.error);
    const otherUsersBooks = res.books.filter(
      (book) => book.ownerId._id !== loggedUser._id
    );
    setBooks(otherUsersBooks);
    books.length === 0 ? setStatus("No books found") : setStatus(`${books.length} book(s) found`)
  };
```
I don't know why it shows that you should delete setPage, of course you dont't :D